admin_guide/persistent_storage/persistent_storage_gce.adoc

```diff
@@ -0,0 +1,128 @@
+= Persistent Storage Using GCE PersistentDisk
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+OpenShift supports GCE PersistentDisk volumes (gcePD). You can provision your OpenShift cluster with
+link:../../architecture/additional_concepts/storage.html[persistent storage] using
+link:https://cloud.google.com/compute/docs/disks/[GCE].
+Some familiarity with Kubernetes and GCE is assumed.
+
+[IMPORTANT]
+====
+Before creating persistent volumes using GCE, OpenShift must first be
+properly link:../configuring_gce.html[configured for GCE PersistentDisk].
+====
+
+The Kubernetes
+link:../../architecture/additional_concepts/storage.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure. Persistent volumes are not bound to a single
+project or namespace; they can be shared across the OpenShift cluster.
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
+volume claims], however, are specific to a project or namespace and can be
+requested by users.
+
+
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+[[gce-provisioning]]
+
+== Provisioning
+Storage must exist in the underlying infrastructure before it can be mounted as
+a volume in OpenShift. After ensuring OpenShift is
+link:../../configuring_gce.html[configured for GCE PersistentDisk], all that is
```

Comment:
Should be `link:../../admin_guide/configuring_gce.html[configured for GCE PersistentDisk]`